response-headers-usecases.md

```diff
@@ -0,0 +1,13 @@
+# Use Cases for response Headers
+
+The [Response Header Specification](https://github.com/w3c/trace-context/blob/master/spec/21-http_response_header_format.md) defines the format for response headers.
+
+As response headers solve (largely) different use cases, this doc aims to collect them and also address possible implications, like privacy issues.
+
+This document is a loose, unordered and ongoing collection.
+
+## Usecase 1: Report the proposed parent ID back to the caller 
```

Comment:
While it is good you describe use cases for such things I believe those should be self described. For example, why some systems can't create a trace ID and span ID, why the caller would need a parent ID? Does the parent ID is enough info without the trace ID?

Use case as it is feels like describing the what and not the why.

Reply:
This topic has been well discussed and is niche at best. It doesn't deserve the "usecase 1" if anything "honorable mention"

Returning trace IDs is far more frequent for putting traces together without this odd notion of "proposed parent ID". Maybe you can put more routine things here by looking at the issues leading up to this or asking users.

For example, this issue from over 4 years ago catalogs some ideas with real people who were interested https://github.com/openzipkin/b3-propagation/issues/4